.tests/cypress/integration/catalog/add-to-wishlist.js

```diff
@@ -0,0 +1,40 @@
+function pickRandomItem(item) {
+  return item[Math.floor(Math.random() * item.length)]
+}
+
+describe('Catalog - Add to Wishlist by logged in user', () => {
+  it('adds a product to wishlist', () => {
+    cy.fixture('urls.json').then(({ men }) => {
+      cy.login()
+      cy.visit(men)
+      cy.waitForCustomerData()
+      cy.get('[data-testid=add-to-wishlist-button]').then(item => {
+        pickRandomItem(item).click()
+      })
+      cy.waitForCustomerData()
+      cy.waitForCartData()
+      // checks product visibility on wishlist page
+      cy.get('[data-row="product-item"]').should('be.visible')
+      cy.get('.wishlist__action').should('be.visible')
+    })
+  })
+
+  it('removes product from wishlist', () => {
+    cy.get('.wishlist__remove').click()
+    cy.get('[data-ui-id="message-success"]').should('be.visible')
```

Comment:
You should check if the wishlist is empty.

Reply:
Removed removing product from wishlist test